### DISCUSSION TITLE
DS%40UMich/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#222] [Announcements]
# DS%40UMich/

None

https://opencs.app/DS%40UMich/

**Comment by shockylove** [comment]
是LSA院下的DS吗，请问信院的MADS，工院IOE的Data Analytics and Applied Statistics 的track怎么样呢

**Reply by YikunHan42** [reply]
这里写的是LSA下的，IOE下面的这个项目没了解过，但是UM IOE很强，貌似是top3
